src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -26,15 +29,15 @@ volatile size_t          sched_cur;
 /* Macro for sinv back to booter from new component */
 enum
 {
-	BOOT_SINV_CAP = round_up_to_pow2(BOOT_CAPTBL_FREE + CAP32B_IDSZ, CAPMAX_ENTRY_SZ)
+	BOOT_SINV_CAP = round_up_to_pow2(BOOT_CAPTBL_FREE + CAP32B_IDSZ, CAPMAX_ENTRY_SZ),
 };
 
 static vaddr_t
 boot_deps_map_sect(spdid_t spdid, vaddr_t dest_daddr)
 {
 	vaddr_t addr = (vaddr_t)cos_page_bump_alloc(&boot_info);
 	assert(addr);
-
+	
```

Comment:
configure vim/emacs to remove blank space on save.  I have the emacs code for this if you need it.